R/reportRenovation.R

```diff
@@ -0,0 +1,105 @@
+    mselect(qty = "area") %>%
+    collapseDim(dim = "qty")
+
+
+
+
+  # REPORT ---------------------------------------------------------------------
+
+  out <- mbind(
+
+    ## Total ====
+    reportAgg(v_renovation,
+              "Renovation|Buildings (bn m2/yr)", brickSets,
+              agg = c(bs = "all", hs = "all", bsr = "all0", hsr = "all", vin = "all", loc = "all", typ = "resCom", inc = "all"),
+              silent = silent),
+    reportAgg(v_renovation,
+              "Renovation|Residential (bn m2/yr)", brickSets,
+              agg = c(bs = "all", hs = "all", bsr = "all0", hsr = "all", vin = "all", loc = "all", typ = "res", inc = "all"),
+              silent = silent),
+    reportAgg(v_renovation,
+              "Renovation|Commercial (bn m2/yr)", brickSets,
+              agg = c(bs = "all", hs = "all", bsr = "all0", hsr = "all", vin = "all", loc = "all", typ = "com", inc = "all"),
+              silent = silent),
+
+
+    ## by building type ====
+    reportAgg(v_renovation,
+              "Renovation|Residential|{typ} (bn m2/yr)", brickSets,
+              agg = c(bs = "all", hs = "all", bsr = "all0", hsr = "all", vin = "all", loc = "all", inc = "all"),
+              rprt = c(typ = "res"),
+              silent = silent),
+
+
+    ## by location ====
+    reportAgg(v_renovation,
+              "Renovation|Residential|{loc} (bn m2/yr)", brickSets,
+              agg = c(bs = "all", hs = "all", bsr = "all0", hsr = "all", vin = "all", typ = "res", inc = "all"),
+              rprt = c(loc = "all"),
+              silent = silent),
+
+
+    ## by initial heating system ====
+    reportAgg(v_renovation,
+              "Renovation|Residential|Initial|{hs} (bn m2/yr)", brickSets,
+              agg = c(bs = "all", bsr = "all0", hsr = "all", vin = "all", loc = "all", typ = "res", inc = "all"),
+              rprt = c(hs = "all"),
+              silent = silent),
+
+
+    ## by building type + initial heating system ====
+    reportAgg(v_renovation,
+              "Renovation|Residential|{typ}|Initial|{hs} (bn m2/yr)", brickSets,
+              agg = c(bs = "all", bsr = "all0", hsr = "all", vin = "all", loc = "all", inc = "all"),
+              rprt = c(hs = "all", typ = "res"),
+              silent = silent),
+
+
+    ## by final heating system ====
+    reportAgg(v_renovation,
+              "Renovation|Residential|Final|{hsr} (bn m2/yr)", brickSets,
+              agg = c(bs = "all", hs = "all", bsr = "all0", vin = "all", loc = "all", typ = "res", inc = "all"),
+              rprt = c(hsr = "all"),
+              silent = silent),
+
+
+    ## by final heating system, without zero state ====
+    reportAgg(v_renovation,
+              "Renovation|Residential|Final with zero|{hsr} (bn m2/yr)", brickSets,
+              agg = c(bs = "all", hs = "all", bsr = "all0", vin = "all", loc = "all", typ = "res", inc = "all"),
+              rprt = c(hsr = "all0"),
+              silent = silent),
+
+
```

Comment:
I would drop this as the zero doesn't have a meaning to anyone who doesn't develop BRICK.
```suggestion
```

Reply:
Long-term I fully agree. But I would suggest to keep it for the time-being, because currently, the people who are using the reporting are actually those who develop Brick ;-)